src/hu/elte/txtuml/stdlib/util/ScannerImplementation.java

```diff
@@ -0,0 +1,77 @@
+package hu.elte.txtuml.stdlib.util;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+
+public class ScannerImplementation implements Scanner{
+	java.util.Scanner sc;
+	
+	@Override
+	public void scanner(File file) {
+		try {
+			sc = new Scanner(file);
+		} catch (FileNotFoundException e) {
+			e.printStackTrace();
+		}
+	}
+
+	@Override
+	public boolean hasNext() {
+		return sc.hasNext();
+	}
+
+	@Override
+	public boolean hasNext(String pattern) {
+		return sc.hasNext(pattern);
+	}
+
+	@Override
+	public boolean hasNextInt(int radix) {
+		return sc.hasNextInt();
```

Comment:
//It's Done .
public boolean hasNextInt(int radix) {
		return sc.hasNextInt(radix);
}